commit a4a12af5abe22d63fbebf0a219d8d13eff6db20c
Author: Carlos O'Donell <carlos@redhat.com>
Date:   Thu Jun 8 07:30:33 2023 -0400

    dirent: Reformat Makefile.
    
    Reflow and sort Makefile.
    
    Code generation changes present due to link order changes.
    
    No regressions on x86_64 and i686.

Conflicts:
	dirent/Makefile (resorting)

diff --git a/dirent/Makefile b/dirent/Makefile
index 5bad3c112209a2ce..450bcd5d8981f40b 100644
--- a/dirent/Makefile
+++ b/dirent/Makefile
@@ -22,17 +22,49 @@ subdir		:= dirent
 
 include ../Makeconfig
 
-headers		:= dirent.h bits/dirent.h bits/dirent_ext.h
-routines	:= opendir closedir readdir readdir_r rewinddir \
-		   seekdir telldir scandir alphasort versionsort \
-		   getdents getdents64 dirfd readdir64 readdir64_r scandir64 \
-		   alphasort64 versionsort64 fdopendir \
-		   scandirat scandirat64 \
-		   scandir-cancel scandir-tail scandir64-tail
-
-tests	   := list tst-seekdir opendir-tst1 bug-readdir1 tst-fdopendir \
-	      tst-fdopendir2 tst-scandir tst-scandir64 \
-	      tst-rewinddir \
+headers := \
+  bits/dirent.h \
+  bits/dirent_ext.h \
+  dirent.h \
+  # headers
+routines := \
+  alphasort \
+  alphasort64 \
+  closedir \
+  dirfd \
+  fdopendir \
+  getdents \
+  getdents64 \
+  opendir \
+  readdir \
+  readdir64 \
+  readdir64_r \
+  readdir_r \
+  rewinddir \
+  scandir \
+  scandir-cancel \
+  scandir-tail \
+  scandir64 \
+  scandir64-tail \
+  scandirat \
+  scandirat64 \
+  seekdir \
+  telldir \
+  versionsort \
+  versionsort64 \
+  # routines
+
+tests := \
+  bug-readdir1 \
+  list \
+  opendir-tst1 \
+  tst-fdopendir \
+  tst-fdopendir2 \
+  tst-rewinddir \
+  tst-scandir \
+  tst-scandir64 \
+  tst-seekdir \
+  # tests
 
 CFLAGS-scandir.c += $(uses-callbacks)
 CFLAGS-scandir64.c += $(uses-callbacks)
